custom/conf/app.example.ini

```
@@ -2487,3 +2487,8 @@ ROUTER = console
 ;PROXY_URL =
 ;; Comma separated list of host names requiring proxy. Glob patterns (*) are accepted; use ** to match all hosts.
 ;PROXY_HOSTS =
+
+;[sync]
+;; lock service type, could be memory or redis
+; LOCK_SERVICE_TYPE = memory
+; LOCK_SERVICE_CONN_STR = ; redis "addrs=127.0.0.1:6379 db=0"
```

```suggestion
;LOCK_SERVICE_TYPE = memory
;LOCK_SERVICE_CONN_STR = ; redis "addrs=127.0.0.1:6379 db=0"
```
Like the other entries. Comments and docs are missing.

---

Done.